macros/SeeCompatTable.ejs

```diff
@@ -12,7 +12,6 @@ var str = mdn.localString({
     "ru"    : "<strong>Это экспериментальная технология</strong><br />Так как спецификация этой технологии ещё не стабилизировалась, смотрите <a href='#Browser_compatibility'>таблицу совместимости</a> по поводу использования в различных браузерах. Также заметьте, что синтаксис и поведение экспериментальной технологии может измениться в будущих версиях браузеров, вслед за изменениями спецификации."
 });
 
-
-%><div class="notice overheadIndicator experimental">
+%><div class="blockIndicator experimental indicator-warning">
```

Comment:
https://github.com/mozilla/kuma/pull/4979 now adds the following indicator classes:
- `.indicator-info` - grey
- `.indicator-version` - blue
- `.indicator-warning` - yellow
- `.indicator-danger` - red